pkg/lib/localstore/ftsearch/jsonhighlighter/json/json.go

```diff
@@ -0,0 +1,89 @@
+//  Copyright (c) 2014 Couchbase, Inc.
```

Comment:
should we have tests for this or is it temporary?

Reply:
this is temporary for the bleve we are going to remove. Also tested as part of queries_test.go